.github/workflows/setup-binary-test.yml

```diff
@@ -43,19 +43,20 @@ jobs:
       - name: 'Checkout'
         uses: 'actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11' # ratchet:actions/checkout@v4
 
-      - name: 'Path'
+      - name: 'OS friendly path'
         id: 'path'
-        shell: 'bash'
+        shell: 'pwsh'
```

Comment:
why are we switching the shell to PowerShell from bash?

Reply:
nvm, switching back to bash.

Reply:
nvm, switched back to powershell but just for windows lol